Cyotek.Windows.Forms.ImageBox/ImageBoxMouseWheelMode.cs

```diff
@@ -0,0 +1,18 @@
+namespace Cyotek.Windows.Forms
+{
+  /// <summary>
+  /// Specifies the way the mouse wheel is handled for the <see cref="ImageBox"/> control.
+  /// </summary>
+  public enum ImageBoxMouseWheelMode
+  {
+    /// <summary>
+    /// Mouse wheel zooms
+    /// </summary>
+    Zoom,
```

Comment:
I usually like my enums to always have a zero value that means do nothing, invalid, etc.

Note that this means you would need to change the constructor to assign the `Zoom` value given it would no longer be a default.

Reply:
Sure, I am going to add this in a follow up commit.